packages/istanbul-reports/lib/html/index.js

```diff
@@ -16,7 +16,8 @@ function htmlHead(details) {
     <meta charset="utf-8" />
     <link rel="stylesheet" href="${html.escape(details.prettify.css)}" />
     <link rel="stylesheet" href="${html.escape(details.base.css)}" />
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" type="image/x-icon" href="https://user-images.githubusercontent.com/903597/69355859-86ff0b80-0c50-11ea-92d5-7ff1997db800.png" />
```

Comment:
Sorry this does not work, these reports must not link to any remove servers.  You need to commit the icon in `packages/istanbul-reports/lib/html/assets/favicon.png` then in `HtmlReport#fillTemplate` you need to add `favicon: linkMapper.assetPath(node, 'favicon.png')`, use `${html.escape(details.favicon)}` to construct the href.

Reply:
Thank you for the detailed instructions on how to resolve. I've amended.